L1Trigger/L1TTrackMatch/plugins/L1TrackJetProducer.cc

```diff
@@ -61,73 +66,69 @@ class L1TrackJetProducer : public stream::EDProducer<> {
 
   // ----------member data ---------------------------
 
+  const EDGetTokenT<vector<TTTrack<Ref_Phase2TrackerDigi_> > > trackToken_;
+  const edm::EDGetTokenT<std::vector<l1t::Vertex> > PVtxToken_;
   vector<Ptr<L1TTTrackType> > L1TrkPtrs_;
-  vector<int> zBinCount_;
   vector<int> tdtrk_;
-  const float trkZMax_;
-  const float trkPtMax_;
-  const float trkPtMin_;
-  const float trkEtaMax_;
-  const float trkChi2dofMax_;
-  const float trkBendChi2Max_;
-  const int trkNPSStubMin_;
-  const double minTrkJetpT_;
-  const double minJetEtLowPt_;
-  const double minJetEtHighPt_;
+  float trkZMax_;
+  float trkPtMax_;
+  float trkPtMin_;
+  float trkEtaMax_;
+  float nStubs4PromptChi2_;
+  float nStubs5PromptChi2_;
+  float nStubs4PromptBend_;
+  float nStubs5PromptBend_;
+  int trkNPSStubMin_;
+  int lowpTJetMinTrackMultiplicity_;
```

Comment:
Why not keep all these const?

Reply:
If I remember correctly, if we put const there we can change the value later. So probably is not the best option for on-the-fly selection